sap-extensions/v3.0-extensions.json

```diff
@@ -35,6 +35,24 @@
         }
       }
     },
+    {
+      "paths": ["definitions.Operation.properties"],
+      "properties": {
+        "x-sap-operation-intent": {
+          "type": "string",
+          "enum": [
+            "create-single",
+            "create-multiple",
+            "read-single",
+            "read-collection",
+            "upsert-single",
+            "upsert-collection",
```

Comment:
You removed "update-single" instead of "update-collection". Probably, copy&paste error.